index.html

```diff
@@ -0,0 +1,306 @@
+</head>
+<body>
+
+    <div id="header-block">
+
+        <!-- Foto/Avatar -->
+        <div id="avatar-block">
+            <img src="./images/meu-avatar.jpeg" alt="avatar" class="avatar">
+            <a href="#" class="avatar-block-name"><h1>Thálisson Gonçalves</h1></a>
+            <p>Full-Stack Developer</p>
+            <p>Analista de Dados</p>
+        </div>
+
+        <div id="scrollable-content">
+            <!-- Endereço e Idade -->
+            <div id="address-block">
+                <div><p>País: <span>Brasil</span></p></div>
+                <div><p>Estado: <span>Minas Gerais</span></p></div>
+                <div><p>Idade: <span>21</span></p></div>
+            </div>
+
+            <hr>
+
+            <!-- Círculo de Progresso Lenguages -->
+            <div class="wrapper-center">
+                <div class="progress-bar">
+                <svg class="progress" data-progress="100" x="0px" y="0px" viewBox="0 0 80 80">
+                    <path class="track" d="M5,40a35,35 0 1,0 70,0a35,35 0 1,0 -70,0" />
+                    <path class="fill" d="M5,40a35,35 0 1,0 70,0a35,35 0 1,0 -70,0" />
+                    <text class="value" x="50%" y="55%">0%</text>
+                    <text class="text" x="50%" y="115%">Português</text>
+                </svg>
+                </div>
+                <div class="progress-bar">
+                    <svg class="progress" data-progress="25" x="0px" y="0px" viewBox="0 0 80 80">
+                    <path class="track" d="M5,40a35,35 0 1,0 70,0a35,35 0 1,0 -70,0" />
+                    <path class="fill" d="M5,40a35,35 0 1,0 70,0a35,35 0 1,0 -70,0" />
+                    <text class="value" x="50%" y="55%">0%</text>
+                    <text class="text" x="50%" y="115%">Inglês</text>
+                    </svg>
+                </div>
+            </div>
+
+            <hr>
+
+            <!-- Barra de Progresso Skills -->
+            <div id="skills-tools-block">
+
+                <h2>Linguagens de Programação:</h2>
+
+                <br>
+
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> JavaScript</p>
+                </div>
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> Python</p>
+                </div>
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> TypeScript</p>
+                </div>
+
+            </div>
+
+            <hr>
+
+            <div id="skills-tools-block">
+
+                <h2>Front-end:</h2>
+
+                <br>
+
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> React</p>
+                </div>
+                <div>
+                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> Web Design Responsivo</p>
+                </div>
```

Comment:
Nao precisa de divs, normalmente usaremos divs quando temos muito conteudos dentro e queremos estilizar como um bloco.

```suggestion
                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> React</p>
                    <p><i class="fa-solid fa-check" style="color: #0ce353"></i> Web Design Responsivo</p>
```

Reply:
Certo, alterações realizadas hehe, thanksss